src/frontend/shortlist/package.json

```diff
@@ -11,6 +11,8 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
   },
   "dependencies": {
+    "@popperjs/core": "^2.11.6",
+    "bootstrap": "^5.2.2",
```

Comment:
I don't think your component needs this?  We didn't add this in yours, it was added in Pooja's

This file should be removed from this PR?